station/setup.py

```diff
@@ -5,31 +5,115 @@
 import stat
 import sys
 
-REQUIREMENTS = [i.strip() for i in open("requirements.txt").readlines()]
+
+def get_requirements_and_links():
```

Comment:
The ```setup.py``` is a standard Python way to install and publish packages.
I notice that in our case it has more disadvantages than advantages as not all our dependencies are in PyPI. Few of them aren't even Python scripts.
I will read a little about how to do it better.

Reply:
I think there's a trade off between turning this into everything-included-with-a-kitchen-sink. I do like the detection of what is installed. Perhaps it should do what brew does on Mac? Prints extra steps necessary to complete, but let the user do it?

Personally, I would focus more on actual reception capabilities. This is never going to be fully automated installation experience. "Some assembly required" will always be part of it.